test/test_utilities.py

```diff
@@ -20,7 +19,9 @@
         pytest.param("2018-12-13", 1544659200),
         pytest.param("2018-12-13T00:00:00-08:00", 1544688000),
         pytest.param("", None),
-        pytest.param("2018-12-13 PST", None, marks=[xfail_value_error]),
+        pytest.param(
+            "2018-12-13 PST", None, marks=[pytest.mark.xfail(raises=ValueError, strict=True)]
+        ),
```

Comment:
this is a nice simplification. was it enforced by one of the new ruff rules? cool if so.

Reply:
If I remember correctly, the linter flagged something else about this code, and then when I was looking at it for simplification, I realized it just didn't need to exist.